### PR TITLE
Switch to local headword retrieval for latin

### PR DESCRIPTION
**Pull request by arthurian**
This PR updates the `LatinService` to use a local headword retrieval strategy rather than calling out to the Perseids Morphology Service (e.g. `MorpheusService`). This is a first step along the path to improving lemmatization and dealing with spelling issues since it gives us more control over the entire process (see also: #350).

Notes:
- Added `latin_lexicon.csv` which maps tokens to lemmas. This is an export of data provided to us by Logeion (`LatinMorph16.db`).
- Added `LatinLexicon` model to `lemmatization.models`. Note that this model includes just the `token` and `lemma` fields from the `LatinMorph16.db`. A `frequency` field has also been added in anticipation of adding frequency data to improve the order of the results when there are multiple matching lemmas, although we don't have that frequency data yet.
- Added `load_latin_lexicon.py` to load the data into `LatinLexicon`, with some minor optimization to do bulk inserts. This is an isolated table that has no relationships with any others.
- Slightly refactored the `services.base` module to include the original `MorpheusService` class, which is now only used by the `GreekService`. 
- Updated `LatinService` to query `LatinLexicon` directly. Note that it matches on the `token` field exactly (for now).
- Added basic unit test in `lemmatization.tests` to exercise the `LatinService`.

@bilbe @Tanvez

### CODE REVIEW

**Review by Tanvez — APPROVED** [review]
Looks good @arthurian! tests and import works great!

File: lemmatization/services/latin.py [hunk 1]
```diff
@@ -34,8 +36,30 @@ def re_tokenize_clitics(tokens):
...
+    def lemmatize(self, form: str) -> List[str]:
+        """
+        Retrieves matching lemmas (headwords) for a word by querying our local database of
+        latin tokens and lemmas. This was previously a remote call to the remote
+        Perseids Morphology Service (e.g. MorpheusService).
+
+        Results are returned in order of highest frequency first, or simply
+        alphabetical by lemma if there is no frequency data.
+        """
+        results_qs = LatinLexicon.objects.filter(token=form)
```
Comment (arthurian) [inline lemmatization/services/latin.py:60]: Should this be a case sensitive search or case insensitive?

**Comment by IvyJL** [comment]
Could we try case sensitive?
Right now I think we are ignoring case somewhere in the process, but in would be nice to distinguish if there are proper names to be found. 
For example, quintus means “fifth” but Quintus is also a male first name. 
>

**Comment by arthurian** [comment]
@IvyJL Yes, absolutely! @bilbe and I were just talking about this and he mentioned something similar. We'll try it case-sensitive for now.

**Comment by bilbe** [comment]
The problem with case sensitive is what about capitalization at the beginning of a sentence or line? If Quintus is capitalized as the first word of a sentence, which is it? Maybe it would be better to give people the option to disambiguate. My two cents.

**Comment by IvyJL** [comment]
Oh, yes, sure. Users may have to disambiguate. But if we could retrieve both upper and lower case so that they could disambiguate, that would be good. Instead of just ignoring all uppercase. I plan to give as best practice in the instructions to only capitalize proper names, which some textbooks do anyway.

**Pull request merged**